### PR TITLE
update definitions and example usage

### PR DESCRIPTION
**Pull request by MaxCorbel**

### CODE REVIEW

**Comment by MaxCorbel** [comment]
Not sure if I am providing too much detail/if we are actually using ISO-3166-1 alpha-2 country code format.

**Review by viv32 — APPROVED** [review]

**Review by PhoenixAlexander-Wright — APPROVED** [review]

File: locations.md [hunk 1]
```diff
@@ -6,13 +6,19 @@
...
-| `id`|bigint|the unique identifier|
-| `name`|string|the name of the location|
-| `country`|string|the country of the location|
+| `id`|int|the unique identifier|
 | `type`|string|the type of location (`market`, `port`, or `zip3`)|
+| `geographyType`|string|the type of geography (`circle` or `none`)|
+| `date`|timestamp|time and date of location creation|
+| `name`|string|the name of the location|
+| `state`|string|the state or province of the location, if applicable|
+| `countryId`|int|the unique identifier of the country
```
Comment (nealsiebert) [inline locations.md:15]: we should omit countyId we don't want people relying on it and in the docs we state to to rely on undocumented api fields

File: locations.md [hunk 2]
```diff
@@ -6,13 +6,19 @@
...
+| `country`|string|the name of the country|
+| `countryCode`|string|the country code in ISO-3166-1-alpha-2 format|
+| `display`|string|display name in format: Country - Name, State|
 
 #### Note(s)
-1. Market locations are used to create international moves, and zip3s are used for CONUS moves.
+1. Market and port locations are used to create international moves and zip3s are used for CONUS moves.
```
Comment (nealsiebert) [inline locations.md:21]: This is only markets, ports are never used to create move

File: locations.md [hunk 3]
```diff
@@ -32,15 +38,27 @@ HTTP/1.1 200 OK
     "data": [
         {
             "type": "market",
-            "id": 1864,
+            "geographyType": "circle",
+            "date": "2022-08-31T18:03:32.000Z",
             "name": "Melbourne",
-            "country": "Australia"
+            "countryId": 4,
```
Comment (nealsiebert) [inline locations.md:44]: omit countryId

File: locations.md [hunk 4]
```diff
@@ -108,15 +140,40 @@ Response:
...
+        "offset": 0,
+        "length": 11
+    }
 }
 ```
 #### Note(s)
-1. For international, the response contains a list of locations sorted by a score. The higher the score, the closer that location is to the given coordinates.
-1. You must specify the type of search. (`zip3` or `lat-long`)
\ No newline at end of file
+1. The mileage displays the exact distance between the input coordinates and the locations coordinates, while the extraMileage displays the distance from the input coordinates to the outermost coordinate considered part of the resulting location.
```
Comment (nealsiebert) [inline locations.md:177]: this should be approximate distance or maybe straight line distance. Really we should just get Miranda to write this sentence for us

**Review by nealsiebert — CHANGES_REQUESTED** [review]

**Pull request merged**